src/bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+// PHP_CodeSniffer does not define an autoloader in its composer.json,
+// so we must invoke it ourselves.
+require __DIR__ . "/../vendor/squizlabs/php_codesniffer/autoload.php";
```

Comment:
Wait, just saw that I do not depend on the unstable version anymore:
```json
        "squizlabs/php_codesniffer" : "^3.0",
```
I wonder how the breaking change got in there then, as `preferStable` is set?
Anyway, if it fixes it and tests pass I would prefer to lock the most recent version that works